pkg/trace/sampler/prioritysampler.go

```diff
@@ -155,8 +195,19 @@ func (s *PrioritySampler) applyRate(sampled bool, root *pb.Span, signature Signa
 	if rate, ok := getMetric(root, deprecatedRateKey); ok {
 		return rate
 	}
-	rate := s.Sampler.GetSignatureSampleRate(signature)
-	if rate > prioritySamplingRateThresholdTo1 {
+	// no rates set in tracers, setting the local rate
+	// if the RemoteRates feedback loop is used for that signature
+	// use the RemoteRates feedback loop rate.
+	if s.remoteRates != nil {
+		rate, ok := s.remoteRates.GetSignatureSampleRate(signature)
```

Comment:
I think it could be clearer like this:
```suggestion
	if s.remoteRates != nil {
		// use a remote rate, if available
		rate, ok := s.remoteRates.GetSignatureSampleRate(signature)
```
I think the rest is obvious from the `if` statements and `return` points in the code.